Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/FindPeakAutomatic.py

```diff
@@ -0,0 +1,292 @@
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+from mantid.simpleapi import *
+from mantid.api import *
+from mantid.kernel import *
+
+import matplotlib.pyplot as plt
+
+import numpy as np
+import scipy.signal
+
+
+class FindPeakAutomatic(PythonAlgorithm):
```

Comment:
This is in the `WorkflowAlgorithms` directory. Is it supposed to be a `DataProcessorAlgorithm` or a regular one?